new-admin/src/index.js

```diff
@@ -18,11 +18,19 @@ import Backbone from "backbone";
 import { Router } from "backbone";
 import ApplicationView from "./views/application.jsx";
 import ApplicationModel from "./models/application.js";
-
+import {
+  initHFetch,
+  hfetch,
+  initFetchWrapper,
+  wrapJqueryAjax,
+} from "./utils/FetchWrapper";
```

Comment:
I haven't noticed until now, but it seem like we have `utils/FetchWrapper` in both `new-admin` and `new-client`. Can't they be consolidated, somehow? I now that we haven't shared code between the different applications before, and perhaps we can wait. But it would be nice not to duplicate and put shared components somewhere global. (On second thought: I guess we should create an issue so this is not forgotten, and just go on as-is. Opinions, @sweco-sesqqq @jesade-vbg?)

Reply:
Yes, this is not optimal. Maybe we could adress this "shared code" issue when creating the new admin? Or separate issue as you wrote.